<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SatDialog</class>
 <widget class="QWidget" name="SatDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Satellites</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QWidget" name="Panel2" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text" stdset="0">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="PRN09">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>PRN09</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="PRN10">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>PRN10</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="PRN11">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>PRN11</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="PRN12">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>PRN12</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="PRN13">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>PRN13</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="PRN14">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>PRN14</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="PRN15">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>PRN15</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="PRN16">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>PRN16</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QWidget" name="Panel3" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text" stdset="0">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="PRN17">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>PRN17</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="PRN18">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>PRN18</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="PRN19">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>PRN19</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="PRN20">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>PRN20</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="PRN21">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>PRN21</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="PRN22">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>PRN22</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="PRN23">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>PRN23</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="PRN24">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>PRN24</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QWidget" name="Panel5" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text" stdset="0">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="PRN25">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>PRN25</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="PRN26">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>PRN26</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="PRN27">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>PRN27</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="PRN28">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>PRN28</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="PRN29">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>PRN29</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="PRN30">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>PRN30</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="PRN31">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>PRN31</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="PRN32">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>PRN32</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QWidget" name="Panel4" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text" stdset="0">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QCheckBox" name="SBAS">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>SBAS</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="GLO">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>GLO</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="GAL">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>GAL</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="PRN33">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>PRN33-</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="Panel1" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text" stdset="0">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="PRN01">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>PRN01</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="PRN02">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>PRN02</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="PRN03">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>PRN03</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="PRN04">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>PRN04</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="PRN05">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>PRN05</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="PRN06">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>PRN06</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="PRN07">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>PRN07</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="PRN08">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>PRN08</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1" colspan="5">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="BtnChkAll">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>&amp;Check All</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnUnchkAll">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>&amp;Uncheck All</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnOk">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>&amp;OK</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnCancel">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>&amp;Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
